Amplify/Categories/DataStore/DataStoreCategory+Behavior.swift

```diff
@@ -54,16 +53,16 @@ extension DataStoreCategory: DataStoreBaseBehavior {
 
     public func delete<M: Model>(_ modelType: M.Type,
                                  withIdentifier id: String,
-                                 where predicate: QueryPredicate?,
-                                 completion: @escaping DataStoreCallback<Void>) where M: ModelIdentifiable,
-                                                                                      M.IdentifierFormat == ModelIdentifierFormat.Default {
+                                 where predicate: QueryPredicate? = nil,
+                                 completion: @escaping DataStoreCallback<Void>)
+    where M: ModelIdentifiable, M.IdentifierFormat == ModelIdentifierFormat.Default {
```

Comment:
let's revert this formatting, the rest of these methods have the format of 
```
completion: @escaping DataStoreCallback<Void> where M: ModelIdentifible, ...
```
instead of 
```
completion: @escaping DataStoreCallback<Void>) 
where M: ModelIdentifiable
```